src/modes/map/map_treasure.cpp

```diff
@@ -318,9 +318,19 @@ void TreasureSupervisor::_UpdateList()
             // If true, the drunes have been selected
             _selection_name.SetText(UTranslate("Drunes"));
             _selection_icon = GlobalManager->Media().GetDrunesIcon();
-            _detail_textbox.SetDisplayText(VTranslate("With the additional %u drunes found in this treasure added, "
-                                                      "the party now holds a total of %u drunes.",
-                                                      _treasure->_drunes, GlobalManager->GetDrunes()));
+
+                // Part 2 of "With the additional %u drune(s) found in this treasure added, ... "
```

Comment:
Indentation is such a nightmare with QTCreator, I have to fix this all up manually in Geany afterwards *headdesk*. It's the only IDE that will run on my Linux machine though, everything else is either too slooow or will crash immediately. How would you feel about using clang-format? We'd need to add a file with formatting instructions for clang-format though.

The comment is for the translators, not for the developers. It will be easier for translators to find with the whole sentence in it.